rune/libcontainer/setns_init_linux.go

```diff
@@ -97,7 +98,12 @@ func (l *linuxSetnsInit) Init() error {
 		if err != nil {
 			return newSystemErrorWithCause(err, "libenclave bootstrap")
 		}
-		return system.Execv("/proc/self/exe", []string{"runelet", "enclave"}, os.Environ())
```

Comment:
Enclave runtime core is still assuming such a re-exec context so we can keep this code if enclave runtime pal used.